tt_metal/distributed/mesh_device.cpp

```diff
@@ -100,8 +100,18 @@ SystemMesh& SystemMesh::instance() {
 }
 void SystemMesh::initialize() {
     this->physical_device_id_to_coordinate = tt::Cluster::instance().get_user_chip_ethernet_coordinates();
-    for (const auto& [chip_id, physical_coordinate] : this->physical_device_id_to_coordinate) {
-        this->physical_coordinate_to_device_id.emplace(physical_coordinate, chip_id);
+    if (this->physical_device_id_to_coordinate.empty()) {
```

Comment:
can we make some checks that this code-path is for BH, GS? Just don't want us to fall into this code-path for invalid configurations on WH.